www/inappbrowser.js

```diff
@@ -115,10 +115,29 @@
         var cb = function (eventname) {
             iab._eventHandler(eventname);
         };
-
+        var strWindowHeaders = '';
+        if (windowHeaders) {
+            if (typeof windowHeaders === 'string' || windowHeaders instanceof String) {
+                strWindowHeaders = windowHeaders.replace(/@/gi, '@a');
```

Comment:
what exactly is the reason for all the replacements of `@` and `=` if you just re-replace it in the corresponding Java/Objective-C code?

Wouldn't it be more readable and clearer to pass in a JSON object and then use Android's included JSONObject support to create a HashMap and iOS' included NSJSONSerialization?

Reply:
The re-replacing is probably meant to keep the headers from being parsed as separate options in the options string.